### PR TITLE
Changes/adds post-apoc prices to the other half of the armor (part 2 of reworking all prices)

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary
```SUMMARY: Balance "adds and changes price values on the other half of the armor"```

#### Purpose of change
The post apocalypse economy and pricing was completely out of whack and needed a full pass, i'm starting work on pricing.
It's going to be hard to compare prices for now because the current prices are so all over the place, so i'm going to have to ask for a little faith that once more of the PR's add up things will click into place and make sense.

#### Describe the solution
adds price-postapoc to all the armors that didn't have them from jewelry.json to undergarment,json

### CODE REVIEW

File: data/json/items/armor/storage.json [hunk 1]
```diff
@@ -731,6 +750,7 @@
     "weight": "840 g",
     "volume": "1500 ml",
     "price": 24000,
+    "price_postapoc": 500,
     "price_postapoc": 36000,
```
Comment (nphyx) [inline data/json/items/armor/storage.json:754]: Duplicate entry here.

File: data/json/items/armor/storage.json [hunk 2]
```diff
@@ -951,6 +975,7 @@
     "weight": "636 g",
     "volume": "3 L",
     "price": 7000,
+    "price_postapoc": 1500,
     "price_postapoc": 19000,
```
Comment (nphyx) [inline data/json/items/armor/storage.json:979]: Another duplicate entry.

**Comment by nphyx** [comment]
Hey, maybe you can fill me in on this - I was using `price_postapoc` to adjust things up or down based on how rare and useful something would be after vs. before the apocalypse (e.g. useful working gadgets that can't be handmade are more expensive, luxury items with no utility beyond wealth signaling are cheaper).

Do you know if that's the right way of going about it, or should post-apoc prices generally be deflationary as you have in this PR?

**Comment by Brian-Otten** [comment]
Thanks for the second pair of eyes, resolved.

**Comment by kevingranade** [comment]
I guess it's leg day.

**Pull request merged**